subsys/net/ip/tcp.c

```diff
@@ -485,24 +485,14 @@ int net_tcp_prepare_segment(struct net_tcp *tcp, u8_t flags,
 		 * have ACK set.
 		 */
 		flags |= NET_TCP_ACK;
-		/* FIXME: We apparently miss increment in another
-		 * transition of the state machine, so have to
-		 * adjust seq no by 2 here. This is required for
-		 * Linux to detect active close on server side, and
-		 * to make Wireshark happy about sequence numbers.
-		 */
-		seq += 2;
+		seq += 1;
```

Comment:
Very minor nit, but this could be `seq++`.

Reply:
Fixed